src/clients/base-evernode-client.js

```diff
@@ -159,8 +161,18 @@ class BaseEvernodeClient {
     }
 
     async #extractEvernodeEvent(tx) {
-        if (!tx.Memos || tx.Memos.length === 0)
+
+        if (!tx.Memos || tx.Memos.length === 0) {
+            if (tx.TransactionType === "NFTokenCreateOffer") {
+                return {
+                    name: EvernodeEvents.NftOfferCreate,
+                    data: {
+                        transaction: tx
```

Comment:
shall we add additional data to the returned data
 `data: {
transaction: tx,
index: tx.index,
tokenId: tx.TokenID,
flag: tx.Flag
}`

Reply:
Ye, I think you guys should for UX reasons.
Users could search the transaction on an explorer to pull `tx.index/tx.TokenID/tx.Flag` but I think providing the data without the user needing to manually search the tx on an explorer would boost the UX.